### ISSUE TITLE
Make it easier for contributors to give their consent 

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutisticaCitizenScience#328]
## Summary

We need to make it easier for people to provide their signature when giving their consent to take part in user testing. This is based on feeedback from an autistic participant who took part in user testing, and wanted to 

## Date Due by

This would be a great thing to do for the first round of user testing when the normal sprint cycle resumes in September. 

## Suggestions for tasks that can fix this issue

There are lots of options for this. Most participants have printed out their consent forms and then scanned them, but this isn't possible for everyone and can be time consuming and awkward.

Docusign might make this easier. 

There are also other options such as using the "draw" tool on Microsoft word or the Whiteboard tool on Zoom. Providing step by step instructions to using these in the emails we send out might help.

**Comment by GeorgiaHCA** [comment]
More feedback:

"Heads up the consent form doesn’t favour completion on an iPad mini or an Android phone. I had to delete the feint white wording in the right fields in order to bring up black ink, then press return to align and after that adjust again to get it to look presentable."
